mcs/class/System/System_test.dll.sources

```diff
@@ -217,7 +217,7 @@ System.Net/DnsTest.cs
 System.Net/EndPointTest.cs
 System.Net/FileWebRequestTest.cs
 System.Net/FileWebResponseTest.cs
-System.Net/FtpWebRequestTest.cs
+#System.Net/FtpWebRequestTest.cs
```

Comment:
Old FTP tests use a simple hand-made FTP Server based on raw sockets but it's not compatible with .NET and .NET Core (both refuse to connect to it due to some protocol violations).
Instead of this test I enabled a corefx one below - tested on my local ftp server, all pass.